Samples/excel-data-types-explorer/README.md

```diff
@@ -0,0 +1,82 @@
+>
+> To try out data types in Office on Windows, you must have an Excel build number greater than or equal to 16.0.14626.10000. To try out data types in Office on Mac, you must have an Excel build number greater than or equal to 16.55.21102600.
+
+## Features
+
+This sample builds and then sideloads a data types explorer add-in that allows you to create and edit data types in an Excel workbook. Once the add-in is sideloaded, you can use it to explore string, double, boolean, entity, web image, and formatted number data types.
+
+![Screenshot showing the data types explorer task pane and a formatted number entity in the Excel grid.](task-pane-data-types-explorer-formatted-number.png)
+
+In particular, this add-in contains an entity data type builder that you can use to create and explore entity cards. To learn more about entity cards, see [Use cards with entity value data types](https://docs.microsoft.com/office/dev/add-ins/excel/excel-data-types-entity-card)
+
+![Screenshot showing the data types explorer task pane, with the entity builder displayed, and an entity card open over the Excel grid.](task-pane-data-types-explorer-entity.png)
+
+## Applies to
+
+- Excel on Windows (minimum build of 16.0.14626.10000)
+- Excel on Mac (minimum build of 16.55.21102600)
+- Excel on the web
+
+## Prerequisites
+
+- Excel on Windows with a minimum build of 16.0.14626.10000), or Excel on Mac with a minimum build of 16.55.21102600), or Excel on the web.
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Create and explore data types in Excel workbooks | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 9-7-2022 | Initial release
+
+----------
+
+## Run the sample
+
+Take the following steps to run this sample and set up the data types explorer add-in.
+
+1. Clone or download this repo.
+1. Navigate to the **Samples/excel-data-types-explorer** folder via the command line.
+1. Run `npm install` to set up the add-in dependencies.
+1. Run `npm install --save @types/office-js-preview` to install the Office.js preview type definitions ([@types/office-js-preview](https://www.npmjs.com/package/@types/office-js-preview)).
+1. Run `npm start`. This command will open Excel, sideload the add-in in Excel, and open the task pane.
+1. Follow the directions in the task pane to build and explore data types.
+
+Another version of this data types explorer tool is available as an [Excel Script Lab sample](https://gist.github.com/mafrenet/e6e1eb26d3ff778edad73a4230b44b5b). To learn more about Script Lab, see [Explore Office JavaScript API using Script Lab](https://docs.microsoft.com/office/dev/add-ins/overview/explore-with-script-lab).
```

Comment:
I'd be hesitant to link to this SL version because that implicitly manes we have to support it as well.

Reply:
I think "See also" is a good suggestion. Maybe frame it as an "unmaintained version" instead of "not supported"? Let's move forward for now and check with David later.